### DISCUSSION TITLE
Translated page and embed code hosted by Github

### DISCUSSION THREAD

**Discussion by innovaweb-dev** [community/community#23429] [New to GitHub]
<p>Can you find a solution for when a page is translated, the embed code will not also translate.<br>
Thanks you GitHub, :+1:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/4/04c588c9ce1b328571f41a802db806529dfddd9d.png" data-download-href="/uploads/short-url/Gd7LnVB0CdnoxeKbEfYjyOQE8d.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/2601347/181076881-30423db7-79ff-441f-8b97-33d0c0a8465d.png" alt="image" data-base62-sha1="Gd7LnVB0CdnoxeKbEfYjyOQE8d" width="690" height="251" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/0/4/04c588c9ce1b328571f41a802db806529dfddd9d_2_690x251.png, https://aws1.discourse-cdn.com/github/optimized/3X/0/4/04c588c9ce1b328571f41a802db806529dfddd9d_2_1035x376.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/0/4/04c588c9ce1b328571f41a802db806529dfddd9d_2_1380x502.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/4/04c588c9ce1b328571f41a802db806529dfddd9d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1418×516 35.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by innovaweb-dev** [comment]
<p>Someone to push up this problem at Github team  ?</p>

**Comment by mgriffin** [comment]
<p>Hi <a class="mention" href="/rifton007">@rifton007</a></p>
<p>I’ve looked into what you have described here and you’re right that embedded code should not get translated automatically like this. I’ve opened up an internal Pull Request to prevent embedded Gists from being translated and I hope that it will get accepted and fixed up soon.</p>
<p>Thanks for bringing this to our attention!</p>
<p>Cheers, Mike.</p>

**Comment by mgriffin** [comment, marked as answer]
<p>Hi <a class="mention" href="/rifton007">@rifton007</a></p>
<p>I wanted to let you know that the change has been made and embedded Gists should not get translated now.</p>
<p>Cheers, Mike.</p>
